src/lib/model/error-codes.ts

```diff
@@ -1,4 +1,6 @@
-// gRPC status (= error) codes, see https://grpc.github.io/grpc/core/md_doc_statuscodes.html
+/*
+ gRPC status (= error) codes, see https://grpc.github.io/grpc/core/md_doc_statuscodes.html
+ */
```

Comment:
Use the correct format to apply the comment to the enum type.
```suggestion
/**
 * gRPC status (= error) codes, see https://grpc.github.io/grpc/core/md_doc_statuscodes.html
 */
```

Reply:
Fixed it.